models/parser/TranslationPurifier.php

```diff
@@ -1,22 +1,24 @@
 <?php
 
-
 namespace humhub\modules\translation\models\parser;
 
-
 use yii\helpers\HtmlPurifier;
 
 class TranslationPurifier extends HtmlPurifier
 {
     /**
      * @inheritDoc
      */
-    public static function configure( $config)
+    public static function configure($config)
     {
-        // https://stackoverflow.com/questions/4566301/htmlpurifier-with-an-html5-doctype
+        // Set HTMLPurifier configuration
         $config->set('HTML.Doctype', 'HTML 4.01 Transitional');
-        $def =  $config->getHTMLDefinition(true);
-        $def->addAttribute('a', 'href', new ParameterURIDef());
-        $def->addAttribute('img', 'src', new ParameterURIDef());
+        $config->set('Attr.EnableID', true);
+
+        // Allow specific tags and attributes
+        $config->set('HTML.Allowed', 'p,b,i,u,s,a[href|target],img[src|alt],ul,ol,li,blockquote,code,pre,span,hr,br,strong');
+        
+        // Allow non-ASCII characters
+        $config->set('Core.EscapeNonASCIICharacters', false);
```

Comment:
And finally see this part of the Docs;
http://htmlpurifier.org/live/configdoc/plain.html#Core.EscapeNonASCIICharacters